src/main/java/com/gmail/chernii/oleksii/YouTubeCommentThread.java

```diff
@@ -0,0 +1,54 @@
+package com.gmail.chernii.oleksii;
+
+import com.google.api.client.auth.oauth2.Credential;
+import com.google.api.services.youtube.YouTube;
+import com.google.api.services.youtube.model.CommentThread;
+import com.google.api.services.youtube.model.CommentThreadListResponse;
+import com.google.common.collect.Lists;
+
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * Created by Space on 07.04.2019.
+ */
+public class YouTubeCommentThread {
+    private static YouTube youTube;
+
+    private String videoID;
+    private long maxSize;
+
+    public YouTubeCommentThread(String videoID, long maxSize) {
+        this.videoID = videoID;
+        this.maxSize = maxSize;
+    }
+
+    private void initialize() {
```

Comment:
firstly public methods then private
[ Ordering of class contents](https://mate-academy.github.io/style-guides/java/java.html#s3.4.2-ordering-class-contents)